openshift/e2e-common.sh

```diff
@@ -154,11 +178,23 @@ function run_conformance_tests(){
       local run_command="-run ^(${test_name})$"
   fi
 
-  go_test_e2e -timeout=30m -parallel=12 ./test/conformance \
+  # check for test flags
+  RUN_FLAGS="-timeout=50m -parallel=20"
```

Comment:
I didn't notice it previously but you're changing parallel to 20 which might cause issues in CI. Please have the default value "-timeout=30m -parallel=12" as you can override it via EVENTING_TEST_FLAGS anyway. Thanks